### PR TITLE
Add code for pinot benchmark as a service

### PR DESCRIPTION
**Pull request by jackjlli**
This PR is the very first step of building a service for Pinot benchmark.

### CODE REVIEW

File: pinot-benchmark/src/main/java/org/apache/pinot/benchmark/PinotBenchmarkServiceStarter.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  private static final Logger LOGGER = LoggerFactory.getLogger(PinotBenchmarkServiceStarter.class);
+
+  private PinotBenchServiceApplication _pinotBenchServiceApplication;
+
+  public PinotBenchmarkServiceStarter() {
+    _pinotBenchServiceApplication = new PinotBenchServiceApplication();
+  }
+
+  public void start() {
+
```
Comment (mcvsubbu) [inline pinot-benchmark/src/main/java/org/apache/pinot/benchmark/PinotBenchmarkServiceStarter.java]: remove extra blank lines. multiple places throughout the file

File: pinot-benchmark/src/main/java/org/apache/pinot/benchmark/PinotBenchmarkServiceStarter.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    LOGGER.info("Stopping Pinot benchmark service...");
+    _pinotBenchServiceApplication.stop();
+  }
+
+
+  public static void main(String[] args) throws InterruptedException {
+    PinotBenchmarkServiceStarter starter = new PinotBenchmarkServiceStarter();
+
+    try {
+      LOGGER.info("Start starter.");
```
Comment (mcvsubbu) [inline pinot-benchmark/src/main/java/org/apache/pinot/benchmark/PinotBenchmarkServiceStarter.java]: Can we improve the log message a bit? 
"Starting benchmark service" works? Similarly for the error log on exception. also, remove the "!" marks

File: pinot-benchmark/src/main/java/org/apache/pinot/benchmark/PinotBenchmarkServiceStarter.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+      starter.start();
+    } catch (Exception e) {
+      LOGGER.error("Error starting!!!", e);
+    }
+
+    int i = 0;
+    while (i++ < 600L) {
+      Thread.sleep(1000L);
+    }
```
Comment (mcvsubbu) [inline pinot-benchmark/src/main/java/org/apache/pinot/benchmark/PinotBenchmarkServiceStarter.java]: Infinite loop?

File: pinot-benchmark/src/main/java/org/apache/pinot/benchmark/PinotBenchmarkServiceStarter.java (hunk 1 above)
Comment (jackjlli) [reply pinot-benchmark/src/main/java/org/apache/pinot/benchmark/PinotBenchmarkServiceStarter.java]: Done.

File: pinot-benchmark/src/main/java/org/apache/pinot/benchmark/PinotBenchmarkServiceStarter.java (hunk 2 above)
Comment (jackjlli) [reply pinot-benchmark/src/main/java/org/apache/pinot/benchmark/PinotBenchmarkServiceStarter.java]: Done.

File: pinot-benchmark/src/main/java/org/apache/pinot/benchmark/PinotBenchmarkServiceStarter.java (hunk 3 above)
Comment (jackjlli) [reply pinot-benchmark/src/main/java/org/apache/pinot/benchmark/PinotBenchmarkServiceStarter.java]: Done.

**Comment by jackjlli** [comment]
@mcvsubbu This is the very first PR for pinot bench service. 
The following logic for cluster management, benchmark execution, etc would be in another PRs based on this PR.

**Comment by npawar** [comment]
Closed this PR due to 6 months inactivity. Reopen if needed.

**Pull request closed without merging**